PolymathNetwork/polymath-apps
# Feature/add docz

- Add Docz to the project:
  How to test:
  - `yarn`
  - `yarn docz:dev`

- Added a build step (open source JS code should be transpiled for people to use)
- Refactored Jest setup

Know issues: at the moment the Docz build breaks when importing anything from the `polymath-ui` package (with a missing dependencies error). So every components that import it are commented.
